src/runtime/bootstrap-lazy.ts

```diff
@@ -32,7 +32,7 @@ export const bootstrapLazy = (lazyBundles: d.LazyBundlesRuntimeData, options: d.
   const customElements = win.customElements;
   const head = doc.head;
   const metaCharset = /*@__PURE__*/ head.querySelector('meta[charset]');
-  const visibilityStyle = /*@__PURE__*/ doc.createElement('style');
+  const dataStyles = /*@__PURE__*/ doc.createElement('style');
```

Comment:
Renamed this since it wasn't really only responsible for visibility styles anymore